app/de/frosner/broccoli/services/NomadService.scala

```diff
@@ -117,43 +119,48 @@ class NomadService @Inject()(configuration: Configuration, consulService: Consul
         jobStatuses = idsAndStatusesWithPeriodic.toMap
         filteredIdsAndStatuses.map { case (key, value) => key }.foreach(requestServices)
       }
-      case Failure(throwable) => {
+      case Failure(throwable: ConnectException) => {
         Logger.error(
           s"Failed to request statuses for ${instanceIds.mkString(", ")} from ${jobsRequest.uri}: $throwable")
         setNomadNotReachable()
       }
+      case Failure(throwable) => {
+        Logger.error(
+          s"Failed to request statuses for ${instanceIds.mkString(", ")} from ${jobsRequest.uri}: $throwable")
```

Comment:
Please log different messages in these branches, as these are essentially different error conditions with different levels of "fatality".  Using different messages allows us to search for each error case in logs.